src/encoded/static/components/objectutils.js

```diff
@@ -948,6 +948,25 @@ export const filterForDatasetFiles = (fileList, datasets) => {
 };
 
 
+/**
+ * Filter an array of files to ones included in the released analyses in `compiledAnalyses`. If
+ * none of the given compiled analyses have the "released" status, filter to the unreleased
+ * analyses files.
+ * @param {array} fileList Files to filter
+ * @param {array} compiledAnalyses Compiled analysis objects
+ * @returns {array} Filtered `fileList`
+ */
+export const filterForReleasedAnalyses = (fileList, compiledAnalyses) => {
+    if (fileList.length > 0 && compiledAnalyses.length > 0) {
```

Comment:
fileList and compiledAnalyses are external to the file and they are in objectutils- Utility class everyone can call. I recommend you do defensive programming and do a falsey check like check for null; so protect the code from bad arguments.